.github/workflows/scripts/slack_notifications/instrumented_gems.txt

```diff
@@ -0,0 +1,59 @@
+activerecord
+activesupport-broadcast_logger
+activesupport-logger
```

Comment:
```suggestion
```

Though there are really similar names, we're not instrumenting these gems. We patch the classes within the ActiveRecord gem. For versions of AR that don't have those classes, we just don't have support.